tests/triggers/test_interval.py

```diff
@@ -48,25 +48,38 @@ def test_total_signals(self):
 
         self.assert_num_signals_notified(1)
 
-    def test_total_signals_2(self):
-        ''' This time we are testing total_signals where it is not a multiple
-        of generate_signals'''
+    def test_extra_generated_signals(self):
```

Comment:
Make sure to include a one-line docstring for every test that describes what the test does. Check out [PEP-257](https://www.python.org/dev/peps/pep-0257/) for more details